src/libcore/macros.rs

```diff
@@ -229,14 +229,26 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer.
+/// Calls `write_fmt` function on a writer
```

Comment:
One question, should it be "Writes" or "Write"?

Some macros "Ensure", some "Asserts". I'm confused